app/Nova/User.php

```diff
@@ -809,4 +813,36 @@ private static function adminCanRun(NovaRequest $request): bool
     {
         return $request->user()->hasRole('admin');
     }
+
+    /**
+     * Helper method: Graduation semester is a 6-digit code by default.
+     * Prepares semester data for display, returning empty if no string/invalid
+     * string is entered.
+     */
+    private static function parseGradSemester(?string $sem): string
```

Comment:
I would gently just not abbreviating semester `$sem` here and naming the variable `$semester` so it's easier for other people to follow later